components/mpas-ocean/src/shared/mpas_ocn_effective_density_in_land_ice.F

```diff
@@ -151,13 +151,13 @@ subroutine ocn_effective_density_in_land_ice_update(forcingPool, &
 #ifdef MPAS_OPENACC
       !$acc parallel loop &
       !$acc    present(effectiveDensityScratch, effectiveDensityCur, &
-      !$acc            landIcePressure, landIceMask, ssh)
+      !$acc            landIcePressure, landIceFloatingMask, ssh)
 #else
       !$omp parallel
       !$omp do schedule(runtime)
 #endif
       do iCell = 1, nCellsAll
-         if (landIceMask(iCell) == 1) then
+         if (landIceFloatingMask(iCell) == 1) then
```

Comment:
This seems right.  The meaning of effective density relies on flotation so if we were to try to use it in the thin film region, we would get a very wrong value because the `ssh` and `landIcePressure` are not in approximate hydrostatic balance.